crates/nargo/tests/test_data/config.toml

```diff
@@ -1,5 +1,7 @@
 # List of tests to be excluded (i.e not run), as their directory name in test_data
-exclude = ["8_integration", "const_fail"]
+# "1_mul", "2_div","3_add","4_sub","5_over", "6","6_array", "7_function","7","8_integration", "assign_ex", "bool_not", "bool_or", "pedersen_check", "pred_eq", "schnorr", "sha256", "tuples"
```

Comment:
Can you remove this commented line?

Reply:
I think the comment is useful, because when I want to exclude some tests, to find the failing one, I copy paste from this comment

Reply:
Alright, I don't have a hard stance against this, as we are still debugging a bunch of things